.envrc

```diff
@@ -0,0 +1,16 @@
+# Automatically sets up your devbox environment whenever you cd into this
+# directory via our direnv integration:
+
+use_devbox() {
+    watch_file devbox.json
+    if [ -f .devbox/gen/shell.nix ]; then
+        DEVBOX_SHELL_ENABLED_BACKUP=$DEVBOX_SHELL_ENABLED
+        use nix .devbox/gen/shell.nix
```

Comment:
Should we ✂️ this line?

Reply:
I'll send a separate PR for that, since it needs to be done in the code generation